src/Raven.Server/ServerWide/Commands/Subscriptions/AcknowledgeSubscriptionBatchCommand.cs

```diff
@@ -76,30 +76,26 @@ protected override BlittableJsonReaderObject GetUpdatedValue(long index, RawData
             {
                 return context.ReadObject(existingValue, SubscriptionName);
             }
-
-            if (IsLegacyCommand())
-            {
-                if (LastKnownSubscriptionChangeVector != subscription.ChangeVectorForNextBatchStartingPoint)
-                    throw new SubscriptionChangeVectorUpdateConcurrencyException($"Can't acknowledge subscription with name {subscriptionName} due to inconsistency in change vector progress. Probably there was an admin intervention that changed the change vector value. Stored value: {subscription.ChangeVectorForNextBatchStartingPoint}, received value: {LastKnownSubscriptionChangeVector}");
-
-                subscription.ChangeVectorForNextBatchStartingPoint = ChangeVector;
-            }
```

Comment:
are we sure we want to remove this? what if we upgraded from 5.2 to 6.0? and we have an ack command which hasn't applied yet?

Reply:
Reverted

Reply:
I think we can do this part of code in case we have a single connection (not concurrent), 
```
                if (LastKnownSubscriptionChangeVector != subscription.ChangeVectorForNextBatchStartingPoint)
                    throw new SubscriptionChangeVectorUpdateConcurrencyException($"Can't acknowledge subscription with name {subscriptionName} due to inconsistency in change vector progress. Probably there was an admin intervention that changed the change vector value. Stored value: {subscription.ChangeVectorForNextBatchStartingPoint}, received value: {LastKnownSubscriptionChangeVector}");
```